challenges/objects-arrays.js

```diff
@@ -50,6 +76,10 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+
+for (let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+}
 console.log(universities)
```

Comment:
To sort the universities, a sort method is needed like this: console.log(universities.sort())